drivers/vulkan/vulkan_context.cpp

```diff
@@ -234,6 +234,43 @@ Error VulkanContext::_create_validation_layers() {
 	return OK;
 }
 
+typedef VkResult (*_vkEnumerateInstanceVersion)(uint32_t *);
+
+void VulkanContext::_obtain_vulkan_version() {
+	// init to Vulkan 1.0
+	vulkan_major = 1;
+	vulkan_minor = 0;
+	uint32_t vulkan_patch = 0; // we don't track this one because we might override our major/minor
```

Comment:
Not sure I understand the "we might override our major/minor" part of the comment.

Reply:
Basically Reduz and I decided that we won't go over Vulkan 1.2 until we start using features. So say that Vulkan 1.3 is defined and returned we're capping it to 1.2 in which case the path number is useless.
But I think the comment is overstated and just confusing so I'll get rid of it. That patch is a local variable and not a class member should make it clear we're not tracking it.